main.go

```diff
@@ -131,10 +131,6 @@ func handleUserRequests(router *mux.Router) {
 	}).Methods(http.MethodPost)
 
 	// Users CRUD Operations
-	router.HandleFunc("/users", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is there an updated API doc cause in the API and Data Reference doc it looks like the user creation endpoint is still there?

Reply:
Whoops... Yep I misread that. Will fix right away!

Reply:
Okay, we can add users again